tests/integration/crossref/test_crossref_views.py

```diff
@@ -83,20 +83,12 @@ def test_crossref_search_handles_the_response_when_the_request_is_valid(log_in_a
 
 
 def test_crossref_search_handles_the_response_when_the_request_asks_for_a_non_existing_doi(log_in_as_scientist, app_client):
-    with requests_mock.Mocker() as requests_mocker:
-        requests_mocker.register_uri(
-            'GET', 'http://api.crossref.org/works/does-not-exist',
-            status_code=404,
-            text=pkg_resources.resource_string(
-                __name__, os.path.join('fixtures', 'does-not-exist')),
-        )
+    response = app_client.get('/doi/search?doi=does-not-exist')
```

Comment:
Here to make sure it does not do a request to crossreff, you can do the `with requests_mocker.Mocker()` thingie but not register any uri, so if it tries it will fail.